sauce/main.js

```diff
@@ -54,4 +59,11 @@ const featureInstances = features.map(Feature => new Feature());
   } else {
     setTimeout(poll, 250);
   }
+  // Check if the passed feature is enabled.
+  function isFeatureEnabled(feature) {
```

Comment:
we'll want to actually move this function to the top of the file next to the `featureInstances` const so we don't re-create it every 250ms :D

Reply:
Doh!